src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -221,7 +221,8 @@
       <LocalizeBinDir>$(OutDir)localize\</LocalizeBinDir>
     </PropertyGroup>
     <ItemGroup>
-      <LocalizedFileToDrop Include="$(LocalizeBinDir)ENU\*.json">
+      <!-- to copy out lcg file -->
+      <LocalizedFileToDrop Include="$(LocalizeBinDir)ENU\*">
```

Comment:
@chuckries No. This is a hack to get the file out. I have it checked in but their tool expects it to be in the drop.